### ISSUE TITLE
Using plugin-angular with Angular 8 Ivy causes errors

### ISSUE THREAD

**Issue by hassanasad** [bugsnag/bugsnag-js#556]
I am using Angular v8.0.1 with `enableIvy` set to `true`. The build fails with the following error:

```
ERROR in There is no format with import statements in '/Users/hassan/project/node_modules/@bugsnag/plugin-angular' entry-point.
```

Can you please check it out.

Versions:
```
Angular CLI: 8.0.3
Node: 12.4.0
OS: darwin x64
Angular: 8.0.1
... animations, cdk, common, compiler, compiler-cli, core, forms
... language-service, material, platform-browser
... platform-browser-dynamic, router, service-worker

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.800.3
@angular-devkit/build-angular     0.800.3
@angular-devkit/build-optimizer   0.800.3
@angular-devkit/build-webpack     0.800.3
@angular-devkit/core              8.0.3
@angular-devkit/schematics        8.0.3
@angular/cli                      8.0.3
@ngtools/webpack                  8.0.3
@schematics/angular               8.0.3
@schematics/update                0.800.3
rxjs                              6.5.2
typescript                        3.4.5
webpack                           4.30.0
```

**Bugsnag:**
```
"@bugsnag/js": "^6.3.0",
"@bugsnag/plugin-angular": "^6.3.0",
```

**Comment by hassanasad** [comment]
Probably a good read: https://github.com/angular/angular/issues/29564

**Comment by bengourley** [comment]
Hi @hassanasad, thanks for the report. We did test with ivy enabled in Angular v7 but I guess something else changed!

We'll look into it.

**Comment by bengourley** [comment]
Please try the prerelease as outlined in https://github.com/bugsnag/bugsnag-js/pull/565#issuecomment-505429174 and confirm whether this solves the issue. Thanks!

**Comment by abigail-bugsnag** [comment]
This was fixed in v6.3.2 (https://github.com/bugsnag/bugsnag-js/releases/tag/v6.3.2)
